tests/installation/partitioning_encrypted_lvm.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use strict;
+use warnings FATAL => 'all';
+use testapi;
+use parent "installbasetest";
+
+sub run {
+    my $self = shift;
+    my $partitioner = $testapi::distri->get_partitioner();
+
+    $partitioner->encrypt_partition_with_too_simple_password(is_lvm => 1);
+
+    $self->assert_true($partitioner->is_encrypted_partition_with_lvm_shown_in_the_list(),
```

Comment:
OOP should allows to create shorter names. Perhaps it is for the level of specialization but it is too long in my opinion.